bluemira/equilibria/fem_fixed_boundary/utilities.py

```diff
@@ -210,6 +210,80 @@ def get_tricontours(x, z, array, value):
     return contours
 
 
+def find_flux_surface(psi_norm_func, psi_norm, mesh=None, n_points=100):
+    """
+    Find a flux surface in the psi_norm function precisely by normalised psi value.
+
+    Parameters
+    ----------
+    psi_norm_func: Callable
```

Comment:
What form should this function take? You can give the function signature as part of the typing. If the function takes an int and a numpy array, and returns a float the signature would be:

```python
Callable[[int, np.ndarray], float]
```

But, further to the typing, a bit more in the docstring about what the inputs to the function should be could be useful.